src/testsuite/arith128_test_i64.c

```diff
@@ -393,102 +452,233 @@ test_vsrad (void)
 #undef __DEBUG_PRINT__
 
 int
-test_permdi (void)
+test_mrghld (void)
 {
   vui64_t i1, i2, e;
   vui64_t j;
   int rc = 0;
 
-  printf ("\ntest_permdi Vector permute doubleword immediate\n");
+  printf ("\ntest_merge Vector Merge High/Low Doubleword\n");
 
-  i1 = (vui64_t )CONST_VINT64_DW(1, 2);
-  i2 = (vui64_t )CONST_VINT64_DW(101, 102);
-  e = (vui64_t )CONST_VINT64_DW(1, 101);
-  j = vec_permdi(i1, i2, 0);
+  i1 = (vui64_t) { 1, 2 };
+  i2 = (vui64_t) { 101, 102 };
+
+  e = (vui64_t) { 1, 101 };
+  j = vec_mrghd (i1, i2);
 
 #ifdef __DEBUG_PRINT__
-  print_v2xint64 ("permdi ( ", i1);
+  print_v2xint64 ("mrghd  ( ", i1);
   print_v2xint64 ("        ,", i2);
-  print_v2xint64 ("     ,0)=", j);
+  print_v2xint64 ("       )=", j);
 #endif
-  rc += check_vuint128x ("vec_permdi:", (vui128_t)j, (vui128_t) e);
+  rc += check_vuint128x ("vec_mrghd :", (vui128_t) j, (vui128_t) e);
 
-  e = (vui64_t )CONST_VINT64_DW(1, 102);
-  j = vec_permdi(i1, i2, 1);
+  e = (vui64_t) { 2, 102 };
+  j = vec_mrgld (i1, i2);
 
 #ifdef __DEBUG_PRINT__
-  print_v2xint64 ("permdi ( ", i1);
+  print_v2xint64 ("mrgld  ( ", i1);
   print_v2xint64 ("        ,", i2);
-  print_v2xint64 ("     ,1)=", j);
+  print_v2xint64 ("       )=", j);
 #endif
-  rc += check_vuint128x ("vec_permdi:", (vui128_t)j, (vui128_t) e);
+  rc += check_vuint128x ("vec_mrgld :", (vui128_t) j, (vui128_t) e);
```

Comment:
(global) space after cast (or not, but it's currently inconsistent)

Reply:
OK, again another nonfunctional change